hrjobcontract/js/test/services/contract-health_test.js

```diff
@@ -9,19 +9,20 @@ define([
   'use strict';
 
   describe('ContractHealthService', function () {
-    var $httpBackend, promise, $rootScope, ContractHealthService;
+    var $httpBackend, $rootScope, promise, ContractHealthService;
 
     beforeEach(module('hrjc'));
-    beforeEach(inject(function (_ContractHealthService_, _$httpBackend_, _$rootScope_) {
+    beforeEach(inject(function (_ContractHealthService_, _$httpBackend_,
```

Comment:
yes, it goes over 80 chars. I could revert it back, I don't think it goes over by much.